tests/integration/targets/test_mysql_role/tasks/mysql_role_initial.yml

```diff
@@ -18,1323 +16,1338 @@
     register: srv
 
   - name: When run with unsupported server versions, must fail
-    <<: *task_params
     mysql_role:
       <<: *mysql_params
       name: test
+    register: result
     ignore_errors: yes
 
   - name: Must fail when meet unsupported version
     assert:
       that:
-      - result is failed
-      - result is search('Roles are not supported by the server')
+        - result is failed
+        - result is search('Roles are not supported by the server')
     when:
-    - srv['version']['major'] < 8
+      - srv['version']['major'] < 8
 
-  # Skip unsupported versions
-  - meta: end_play
+  - name: Skip unsupported versions
+    meta: end_play
     when: srv['version']['major'] < 8
 
   #########
   # Prepare
-  - name: Create db {{ test_db }}
-    <<: *task_params
+  - name: Create db test_db
     mysql_db:
       <<: *mysql_params
-      name: '{{ test_db }}'
+      name: 'test_db'
+    register: result
```

Comment:
Registering results to be overwritten in the next task is unnecessary, what's the rationale?